### ISSUE TITLE
Using npm and traditional node, cannot use because of it forcing es6.

### ISSUE THREAD

**Issue by spunkedy** [a-ignatov-parc/tvdml#7]
Just a heads up. I don't think there is an es5 version being built for npm. So by doing:

```
npm install --save tvdml
```

and having a file of:
```
var TVDML = require('tvdml');

console.log(TVDML);
```

I get this error:
```
/Users/firl/Desktop/scratch/testtvdml/node_modules/tvdml/src/index.js:1
(function (exports, require, module, __filename, __dirname) { import * as hooks from './navigation/hooks';
                                                              ^^^^^^
SyntaxError: Unexpected token import
    at Object.exports.runInThisContext (vm.js:76:16)
    at Module._compile (module.js:528:28)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/firl/Desktop/scratch/testtvdml/test.js:1:75)
    at Module._compile (module.js:556:32)
```

This is just a heads up since the README mentions that I should be able to do that. Just started using the library, quite enjoying it.

**Comment by a-ignatov-parc** [comment]
Thank you! Totally forget to add `dist` folder to npm package. Will fix it in nearest days ;)

**Comment by a-ignatov-parc** [comment]
Published `v1.0.1` to npm in order to fix this.
